pkg/cmd/cli/cmd/create/route.go

```diff
@@ -85,7 +85,7 @@ func NewCmdCreateEdgeRoute(fullName string, f *clientcmd.Factory, out io.Writer)
 	cmd.MarkFlagFilename("key")
 	cmd.Flags().String("ca-cert", "", "Path to a CA certificate file.")
 	cmd.MarkFlagFilename("ca-cert")
-	cmd.Flags().String("wildcardpolicy", "", "Sets the WilcardPolicy for the hostname, the default is \"None\". valid values are \"None\" and \"Subdomain\"")
+	cmd.Flags().String("wildcard-policy", "", "Sets the WilcardPolicy for the hostname, the default is \"None\". valid values are \"None\" and \"Subdomain\"")
```

Comment:
if default value is `None`, then it should be the second parameter here as well: `...String("wildcard-policy", "None", "Sets...")`

Also, s/WildcardPolicy/wildcard policy